lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/sendtext'
+
+class Restaurant
+  attr_reader :menu, :sendtext
+
+  def initialize (sendtext = SendText.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)